src/http/server.rs

```diff
@@ -689,6 +689,11 @@ impl<'a> EspHttpConnection<'a> {
         self.headers.is_none()
     }
 
+    #[inline(always)]
```

Comment:
```suggestion
    #[inline]
```

Let let the optimizer do its job :D.